ws/v2.md

```diff
@@ -122,3 +124,5 @@ Updates can be sent in bulk since the payload of the message specifies a collect
 ```
 
 *   `requestId` - a string uuid that corresponds to the requestId sent by the client in the `subscribe` message
+*   `timestamp` - a mandatory 64-bit integer depicting the last time this order was updated. If this is a new order it represents when the order was first signed. Its format is UTC (UNIX Epoch time in milliseconds).
```

Comment:
nit: "mandatory" is a bit strong since the whole "metaData" object is optional.

Reply:
ok, it really makes it hard to show the latest state of the order as fast as possible to the whole liquidity pool if the relayers don't put this information on. if it's not provided the aggregators can't release the information until it's determined from the blockchain.